tests/api/features/Authentication.feature

```diff
@@ -0,0 +1,13 @@
+Feature:
+    In order to access the API
+    As a visitor
+    I want to login to the API
+
+    Background:
+        Given a running instance of Centreon Web API
+
+    Scenario:
```

Comment:
Kindly, could you add the succesful connection test too ?

Reply:
I discuss about this with @kduret, succesful connection test are already handle into the ApiContext : Given I am logged in